backend/scheduler/src/jobs/auto-award-bounty.ts

```diff
@@ -82,9 +81,9 @@ The following comments have been submitted:
 ` + sortedComments.map((c) => `${c.likes} likes:\n${c.content}`).join('\n\n')
     const resultMessage = await promptGPT4(prompt)
     if (resultMessage) {
-      const firestore = admin.firestore()
-      const contractRef = firestore.collection('contracts').doc(contract.id)
-      await contractRef.update({ gptCommentSummary: resultMessage })
+      await updateContract(pg, contract.id, {
```

Comment:
I don't see the implementation for `updateContract`. This PR is missing the `contracts.ts` file?

It should be broadcasting on the websocket, correct?